src/components/attendee/ProfileOverlay/ProfileOverlay.tsx

```diff
@@ -200,11 +217,19 @@ export const ProfileOverlay: React.FC<ProfileOverlayProps> = ({ profile }) => {
             >
               {isSubmitting ? "Saving..." : "Save changes"}
             </Button>
-            <Button disabled={isSubmitting} variant="alternative">
+            <Button
+              disabled={isSubmitting}
+              variant="alternative"
+              onClick={onOverlayClose}
+            >
               Cancel
             </Button>
             {isSuccess ? <span>✓ Saved</span> : ""}
           </div>
+
+          <Button variant="alternative" onClick={logout} large>
```

Comment:
Will this make the flag prop more visible?

```suggestion
          <Button large variant="alternative" onClick={logout}>
```

Reply:
moved: https://github.com/sparkletown/sparkle/pull/3003/commits/07ec924f308486ffff85bb746fdba69810c19e56